tests/sentry/deletions/test_group.py

```diff
@@ -182,32 +190,87 @@ def test_delete_groups_delete_grouping_records_by_hash(
 
 
 class DeleteIssuePlatformTest(TestCase, SnubaTestCase, OccurrenceTestMixin):
-    def test_issue_platform(self):
-        event = self.store_event(data={}, project_id=self.project.id)
-        issue_occurrence, group_info = self.process_occurrence(
+    referrer = "testing.test"
+
+    def create_occurrence(
+        self, event: Event, type_id: int
+    ) -> tuple[IssueOccurrence, GroupInfo | None]:
+        occurrence, issue_platform_group = self.process_occurrence(
             event_id=event.event_id,
-            project_id=self.project.id,
-            # We are using ReplayDeadClickType as a representative of Issue Platform
-            type=ReplayDeadClickType.type_id,
-            event_data={
-                "fingerprint": ["issue-platform-group"],
-                "timestamp": before_now(minutes=1).isoformat(),
-            },
+            project_id=event.project.id,
+            type=type_id,
+            event_data={},
         )
+        return occurrence, issue_platform_group
+
+    def select_issue_platform_events(self, project_id: int) -> object:
+        columns = ["event_id", "group_id", "occurrence_id"]
+        return self.select_rows(Entity(EntityKey.IssuePlatform.value), columns, project_id)
+
+    def select_rows(self, entity: Entity, columns: list[str], project_id: int) -> object:
+        # Unfortunatelly, the cache is always used when we call bulk_snuba_queries even if we pass use_cache=False
```

Comment:
fyi, use_cache is a cache on our side. Snuba also has its own cache that we can't skip